### PR TITLE
Add prompt suggestions for the Qiskit Code Assistant

### PR DESCRIPTION
**Pull request by frankharkins**
To support the new Qiskit Code Assistant, we're adding some prompt suggestions in relevant places in the docs. These prompts produced decent results most of the time.

This is ready for review, but can't be merged until the new component is ready.

### CODE REVIEW

File: docs/guides/configure-error-suppression.mdx [hunk 1]
```diff
@@ -38,6 +38,11 @@ sampler.options.dynamical_decoupling.sequence_type = "XpXm"
 print(f">>> dynamical decoupling sequence to use: {sampler.options.dynamical_decoupling.sequence_type}")
 ```
 
+<CodeAssistantAdmonition
```
Comment (frankharkins) [inline docs/guides/configure-error-suppression.mdx:41]:
```suggestion
<CodeAssistantAdmonition>
```
oops

File: docs/guides/transpile-with-pass-managers.ipynb [hunk 2]
```diff
@@ -102,6 +102,13 @@
    "source": [
     "See [Transpilation defaults and configuration options](defaults-and-configuration-options) for a description of the possible arguments to the `generate_preset_pass_manager` function. The arguments to `generate_preset_pass_manager` match the arguments to the [`transpile`](/api/qiskit/compiler#qiskit.compiler.transpile) function.\n",
     "\n",
+    "<CodeAssistantAdmonition>\n",
+    "Bad at remembering? Try asking the Qiskit Code Assistant.\n",
```
Comment (abbycross) [inline docs/guides/transpile-with-pass-managers.ipynb:106]: 🤣 I love "bad at remembering" but I suggest toning it down a bit to something like "Having trouble remembering?"

File: docs/guides/construct-circuits.ipynb [hunk 3]
```diff
@@ -460,6 +460,17 @@
     "qc.parameters"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "ae19e48d",
+   "metadata": {},
+   "source": [
+    "<CodeAssistantAdmonition>\n",
+    "Forgotten the method name? Try asking the Qiskit Code Assistant!\n",
```
Comment (abbycross) [inline docs/guides/construct-circuits.ipynb:469]:
```suggestion
    "Forgotten the method name? Try asking the Qiskit Code Assistant.\n",
```

File: docs/guides/install-qiskit.mdx [hunk 4]
```diff
@@ -139,6 +139,13 @@ If you are planning to work locally and use simulators built into Qiskit, then y
     Periodically check the [Qiskit release notes](../api/qiskit/release-notes) and the [Qiskit Runtime release notes](../api/qiskit-ibm-runtime/release-notes) to see new releases. We recommend frequently updating your requirements for `qiskit` and `qiskit-ibm-runtime` by, for example, changing the versions in `requirements.txt` to the latest versions, then running `pip install -r requirements.txt` or the appropriate command for your dependency management workflow.
 </Admonition>
 
+<CodeAssistantAdmonition>
+Need help? Try asking the Qiskit code assistant.
```
Comment (abbycross) [inline docs/guides/install-qiskit.mdx:143]:
```suggestion
Need help? Try asking the Qiskit Code Assistant.
```

File: docs/guides/plot-quantum-states.ipynb [hunk 5]
```diff
@@ -446,7 +446,13 @@
     "- **title** (str): a string for the plot title, displayed at the top of the plot\n",
     "- **figsize** (tuple): figure size in inches (width, height)\n",
     "\n",
-    "The `plot_state_city` and `plot_state_paulivec` functions also accept a **color** argument (list of strings) specifying the colors of the bars. See the [API documentation](/api/qiskit/visualization) for more information."
+    "The `plot_state_city` and `plot_state_paulivec` functions also accept a **color** argument (list of strings) specifying the colors of the bars. See the [API documentation](/api/qiskit/visualization) for more information.\n",
+    "\n",
+    "<CodeAssistantAdmonition>\n",
+    "Can't remember the name of the plotting function you need? Try asking the Qiskit Code Assistant!\n",
```
Comment (abbycross) [inline docs/guides/plot-quantum-states.ipynb:452]:
```suggestion
    "Can't remember the name of the plotting function you need? Try asking the Qiskit Code Assistant.\n",
```

File: docs/guides/save-jobs.ipynb [hunk 6]
```diff
@@ -106,6 +106,16 @@
     "retrieved_job.result()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<CodeAssistantAdmonition>\n",
+    "Always forgetting how to retrieve a job? Try this prompt with the Qiskit Code Assistant!\n",
```
Comment (abbycross) [inline docs/guides/save-jobs.ipynb:114]:
```suggestion
    "Always forgetting how to retrieve a job? Try this prompt with the Qiskit Code Assistant.\n",
```

**Review by abbycross — APPROVED** [review]
Couple of small tweaks, otherwise looks good!

File: docs/guides/transpile-with-pass-managers.ipynb (hunk 2 above)
Comment (frankharkins) [reply docs/guides/transpile-with-pass-managers.ipynb:106]:
```suggestion
    "Having trouble remembering pass manager details? Try asking the Qiskit Code Assistant.\n",
```

**Review by cbjuan — APPROVED** [review]
LGTM. Thanks @frankharkins !

**Review by Eric-Arellano — APPROVED** [review]

**Comment by Eric-Arellano** [comment]
Looks beautiful: https://qiskit.github.io/documentation/pr-1918/guides/configure-error-suppression

Feel free to merge once this goes green

**Pull request merged**